util/ident.rb

```diff
@@ -34,6 +34,10 @@ def main
       o.bool '-d', '--debug', 'enable debug mode'
       o.bool '-l', '--list', 'list all checks'
 
+      o.bool '-n', '--noisy', 'noisy fingerprinting.'
+
+      o.string '-ft', '--filter-by-tags', 'filters by tags. ex -ft "saas,iot"'
```

Comment:
lets have the short version be `-w` and the long `--checks-with-tag`. I think this is better than "filter", since we're not filtering anything really. What we're actually doing is running checks with the specified tags ;)